spack.yaml

```diff
@@ -2,24 +2,29 @@
 #
 # It describes a set of packages to be installed, along with
 # configuration settings.
+
+# Build with:
+# - UM7 from dev-access-esm1.6
+# - MOM5 from ESM1.5
+# - CICE4 from ESM1.5 and change to CICE5 once CICE5 SPR has been updated.
 spack:
   specs:
-    - access-esm1p6@git.2024.10.0
+    - access-esm1p6@git.dev_2024.12.0 ~generic-tracers
```

Comment:
Sorry if we covered this already, but why isn't the variant specified in the `mom5` package requirements?

Reply:
The SBR contains:
```
    # TODO: Use the access-esm1.6 branch when it has been confirmed that
    #       MOM5 master supports building with generic tracers disabled.
    depends_on(
        "mom5@access-esm1.5 type=ACCESS-CM",
        type="run",
        when="~generic-tracers",
    )
    depends_on(
        "mom5@access-esm1.6 type=ACCESS-ESM",
        type="run",
        when="+generic-tracers",
    )
```

Reply:
You see, I knew there was a reason.